src/components/explorer-page/main-bounty-card/index.tsx

```diff
@@ -0,0 +1,43 @@
+import Chip from 'components/common/chip';
+import NavigateNextIcon from '@mui/icons-material/NavigateNext';
+const BountyCard = CardProps => (
```

Comment:
Could you include a type definition for `CardProps`? In its current form, it's not really intuitive how to use the `BountyCard` component.

Reply:
Ok